lib/record_store/record.rb

```diff
@@ -56,6 +56,19 @@ def key
       "#{type},#{fqdn}"
     end
 
+    def rdata
+      raise NotImplementedError
+    end
+
+    def rdata_txt
+      raise NotImplementedError
+    end
+
+    def to_s
+      rr_type = self.class.name.demodulize
+      "[#{rr_type}Record] #{fqdn} #{ttl} IN #{rr_type} #{rdata_txt}"
```

Comment:
Also... maybe we could remote one of the `rr_type` from this string. Not a big deal though.

Reply:
So I have `[#{rr_type}Record]` to make it easier to parse the logs during deploys. I'm happy to just have the records match what you'd see coming out of a zone file. I don't have a strong opinion either way. Happy to change if Edgescale prefers the zone file version 🙂

Reply:
That's just fine for now.